bpf/lib/nodeport.h

```diff
@@ -1932,6 +1949,14 @@ static __always_inline int rev_nodeport_lb4(struct __ctx_buff *ctx, int *ifindex
 	ret = ct_lookup4(get_ct_map4(&tuple), &tuple, ctx, l4_off, CT_INGRESS, &ct_state,
 			 &monitor);
 
+	/* rev_nodeport_lb4 function called for all outboud traffic
+	 * at host interface and
+	 * if traffic is not NodePort, return with success.
+	 */
+	if (is_revnat_at_egress_host && !ct_state.node_port) {
+		return CTX_ACT_OK;
+	}
```

Comment:
nit: coding style, pls remove the {} for just single statement

Reply:
sure